README.rst

```diff
@@ -1095,6 +1096,16 @@ Choose from the list of available rules:
   Each namespace use MUST go on its own line and there MUST be one blank
   line after the use statements block.
 
+* **single_line_comment_style**
+
+  Converts multi-line comments that have only one line of actual content
+  into single-line comments, and hash comments to slash.
+
+  Configuration options:
+
+  - ``comment_type`` (``'all'``, ``'hash'``, ``'star'``): fix star comment ``/* */`` [``star``],
```

Comment:
IMO this option should be an array that can contain a subset of `['star', 'hash']`. `'all'` shouldn't be allowed as it would enable new behavior added in the future by default (very unlikely, but still).

But I think boolean options would be fine as well.